README.md

```diff
@@ -202,7 +202,14 @@ It will generate the built application in the `dist` folder.
 
 To build this app with docker, please use this command:
```

Comment:
add explanation on this Dockerfile will build a leto version with two main plugin (terraform/github)